File: vision-quickstart/app/src/main/java/com/google/mlkit/vision/demo/preference/fragment/SettingFragment.java
package com.google.mlkit.vision.demo.preference.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.google.mlkit.vision.demo.R;
import com.google.mlkit.vision.demo.preference.adapter.SettingsAdapter;
import com.google.mlkit.vision.demo.preference.entity.SettingEntity;

import java.util.ArrayList;

public class SettingFragment extends Fragment {
    private View inflate;
    private RecyclerView settingRecy;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        inflate = inflater.inflate(R.layout.fragment_setting, container, false);
        settingRecy = inflate.findViewById(R.id.settingRecy);
        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(requireActivity());
        linearLayoutManager.setOrientation(LinearLayoutManager.VERTICAL);
        settingRecy.setLayoutManager(linearLayoutManager);

        ArrayList<SettingEntity> list = new ArrayList<>();
        list.add(new SettingEntity(R.drawable.icon1, "Update database"));
        list.add(new SettingEntity(R.drawable.systemupdate, "Bluetooth"));
        list.add(new SettingEntity(R.drawable.icon, "Data usage"));
        list.add(new SettingEntity(R.drawable.more, "More"));
        list.add(new SettingEntity(R.drawable.group2, "Display"));
        list.add(new SettingEntity(R.drawable.shape, "Sound & notification"));
        list.add(new SettingEntity(R.drawable.group1, "Apps"));

        SettingsAdapter settingAdapter = new SettingsAdapter(list);
        settingRecy.setAdapter(settingAdapter);

        return inflate;
    }
}

File: vision-quickstart/app/src/main/java/com/google/mlkit/vision/demo/preference/fragment/SettingFragment.kt
package com.google.mlkit.vision.demo.preference.fragment

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.google.mlkit.vision.demo.R
import com.google.mlkit.vision.demo.preference.adapter.SettingsAdapter
import com.google.mlkit.vision.demo.preference.entity.SettingEntity

class SettingFragment : Fragment() {
    var inflate: View? = null
    var settingRecy:RecyclerView? = null

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        inflate = inflater.inflate(R.layout.fragment_setting, container, false)
        settingRecy = inflate!!.findViewById<RecyclerView>(R.id.settingRecy)
        var linearLayoutManager = LinearLayoutManager(activity!!)
        linearLayoutManager.orientation = LinearLayoutManager.VERTICAL
        settingRecy!!.layoutManager = linearLayoutManager
        var list = ArrayList<SettingEntity>()
        list.add(SettingEntity(R.drawable.icon1,"Update database"))
        list.add(SettingEntity(R.drawable.systemupdate,"Bluetooth"))
        list.add(SettingEntity(R.drawable.icon,"Data usage"))
        list.add(SettingEntity(R.drawable.more,"More"))
        list.add(SettingEntity(R.drawable.group2,"Display"))
        list.add(SettingEntity(R.drawable.shape,"Sound & notification"))
        list.add(SettingEntity(R.drawable.group1,"Apps"))



        var settingAdapter = SettingsAdapter(list)
        settingRecy!!.adapter = settingAdapter


        return inflate
    }
}